nexus/examples/qmcpack/rsqmc_misc/estimators/iron_ldaU_dmc.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+
+from nexus import settings,job,run_project
+from nexus import generate_physical_system,read_structure
+from nexus import generate_pwscf
+from nexus import generate_pw2qmcpack
+from nexus import generate_qmcpack,vmc,dmc
+from nexus import ppset,obj
+
+settings(
+    pseudo_dir    = './pseudopotentials',
+    results       = '',
+    sleep         = 3,
+    machine       = 'ws16',
+    status_only   = 0,
+    generate_only = 1,
+    )
+
+ppset(
+    label   = 'ccECP-soft',
+    pwscf   = 'Fe.ccECP-soft.upf'.split(),
+    qmcpack = 'Fe.ccECP-soft.xml'.split(),
+    )
+pseudos = 'ccECP-soft'
+
+# inputs shared by all dft calculations
+dft_shared = obj(
+    input_type       = 'generic',
+    input_dft        = 'lda',
+    ecutwfc          = 400,
+    bandfac          = 1.5,    # Adjust as needed for the 1RDM orbital basis
+    occupations      = 'smearing',
+    smearing         = 'fermi-dirac',
+    degauss          = 1e-3,
+    mixing_beta      = 0.4,
+    nspin            = 2,
+    U_projection_type = 'ortho-atomic',
+    hubbard_u        = obj(Fe=2.0),
```

Comment:
Should be updated for compatibility with newer QE versions.  See Kayahan's PR.

Reply:
Somehow, I can't get it working with the new QE input style for DFT+U. Left @kayahans a message.

Reply:
This is also fixed now.